m4_sessions/1711-inning/Group_7.md

```diff
@@ -21,23 +21,40 @@ Turing and life are hard enough. If you are a minority, introverted, or just ine
-(one station per facilitator)
-
-Hector will facilitate journaling and final reflection (we’ll all come up with journal prompts and have a synopsis of what to tell them to share for final reflection)
-6. Journaling- 5 min
-7. Final reflection- optional individuals shares takeaways - 10min
+#### Intro - Lola - 4 min
+- (College Humor Video)[https://www.youtube.com/watch?v=0lDRz8qmXpE]
+
+#### IceBreaker - Lola - 15 min
+- Whip Around
+-- What is a common misconception about you? || Tell us about how you can relate to the video
+-- Read article https://www.psychologytoday.com/blog/happiness-purpose/201307/speak-yourself
+
+#### Personal Story - Amy - 5 min
+- Amy’s story
+-- Amy gives the synopsis of her experience with the need to speak up for herself at Turing
+
+#### How to Speak Up - Luke - 10 min
+- (Hook Video Clip)[https://www.youtube.com/watch?v=MEDgtjpycYg] (10 min)
+	if it’s past 1:25pm, skip video (last 7 mins)
+
+#### Scenario Rotation - Luke, all - 26 min
+- Scenarios and discussion
+- We’ll have four stations w/ a scenario for each and collect ideas on each scenario to share at the end of the exercise 
+   
+1. You're in your future job. The senior dev you are pairing with refuses to truly listen to your input. You have tried to suggest things to console log, debug, etc. but the response is constantly something like, "No, that won't work." What can you do to navigate this?
+2. You are paired for a week-long project with a classmate.
+- A few days in, your partner starts failing to honor some of the commitments made in the DTR (coming to work time late, leaving early with little notice, etc.). 
+- You commited to working on a certain (difficult) feature together, but you find them in the classroom working on it alone. You ask them to plug in, and they say they'd rather work alone. You are worried that you won't learn this piece of technology as well because you weren't able to be a part of writing this code. 
+3. During an open discussion Posse/Gear Up session, one individual monopolizes the conversation. You have something to add to the conversation, but don't feel there is an 'in' to speak up.
+4. You are paired with a classmate for a week-long project. The partnership is not balanced - on technical understanding/confidence in technical ability - the project is not progressing and one partner doesn't feel safe to ask questions about the project.
+
+#### Closing - Hector - 15 min
+- Journaling (5 min)
+- Final reflection (10 min) 
+1. Think back to a time at Turing where you felt like you could not advocate for yourself. How would you act different with the knowledge you have now?
+2. In reflection, is there a time at Turing where you may have created an environment that made it difficult or challenging for someone else to speak up for themselves?
+3. Is there a strategy or phrase that you have added to your toolkit if/when you run into a challenging pairing in the future?
+4. Another other takeaways.
```

Comment:
Are you going to share whole group?  Maybe you could close by providing some of the specific ideas from the article/video as a summary.